modules/javafx.graphics/src/main/java/javafx/scene/Node.java

```diff
@@ -1950,8 +1950,17 @@ private void updateDisabled() {
      * into the branch until it finds a match. If more than one sub-node matches the
      * specified selector, this function returns the first of them.
      * <p>
+     *     If the lookup selector does not specify a pseudo class, the lookup will ignore pseudo class states;
+     *     it will return the first matching node whether or not it contains pseudo classes.
+     * </p>
+     * <p>
```

Comment:
You don't need to surround a paragraph with `<p>...</p>`, a single `<p>` (without a corresponding `</p>`) between two paragraphs is sufficient. This reduces nesting, as it keeps the subsequent paragraphs on the same indentation.